config/remote/remote.go

```diff
@@ -0,0 +1,205 @@
+package remote
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"sync"
+
+	"github.com/NordSecurity/nordvpn-linux/internal"
+)
+
+type RemoteStorage interface {
+	GetRemoteFile(name string) ([]byte, error)
+}
+
+// ConfigGetter get values from remote config
+type ConfigGetter interface {
+	GetTelioConfig() (string, error)
+	FeatureConfig
+}
+
+type FeatureConfig interface {
+	IsFeatureEnabled(featureName string) bool
+	GetFeatureParam(featureName, paramName string) (string, error)
+}
+
+type ConfigLoader interface {
+	LoadConfig() error
+}
+
+var (
+	envUseLocalConfig = "USE_LOCAL_CONFIG"
+)
+
+type CdnRemoteConfig struct {
+	appVersion     string
+	appEnvironment string
+	localCachePath string
+	remotePath     string
+	cdn            RemoteStorage
+	Features       FeatureMap
+	mu             sync.Mutex
```

Comment:
`LoadConfig` is called as a job, may this introduce some random latency during `GetFeatureParam`/`IsFeatureEnabled`/`GetTelioConfig`? Maybe `RWMutex` would be safer?